sdk/search/azure-search-documents/azure/search/documents/_internal/aio/_search_client_async.py

```diff
@@ -45,6 +47,9 @@ class SearchClient(HeadersMixin):
     def __init__(self, endpoint, index_name, credential, **kwargs):
         # type: (str, str, AzureKeyCredential, **Any) -> None
 
+        api_version = kwargs.pop('api_version', None)
+        if api_version:
+            check_api_version(api_version)
```

Comment:
I see a lot the pattern "if api_version: check", does it mean check should not raise if api_version is None? And then simplify all those occurences?